### PR TITLE
Updated PPP (dave)

### PR DESCRIPTION
**Pull request by davelowqx**
nothing much to write about...

### CODE REVIEW

File: docs/team/davelowqx.md [hunk 1]
```diff
@@ -1,52 +1,29 @@
...
 Given below are my contributions to the project.
 
-- **New Feature**: Added the ability to undo/redo previous commands.
+- **New Feature**: Adding Events
 
-  - What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  - Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  - Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  - Credits: _{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}_
+  - What it does: Adds an Event to Managera.
```
Comment (lowkangn) [inline docs/team/davelowqx.md:15]: You can write a lot more than this for the new feature, don't sell yourself short! 😄 
It is a core feature so there should be plenty that you can write about it. If you want you can look at some of the PPPs that the rest of us have written to get an idea. You can elaborate more of how a user would go about using this feature and why they would need it.

File: docs/team/davelowqx.md [hunk 2]
```diff
@@ -1,52 +1,29 @@
...
+  - Justification: Core functionality for Managera to manage events.
 
-- **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+- **New Feature**: Deleting Events
 
-- **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=AY2122S1-CS2103T-T10-2&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=davelowqx&tabRepo=AY2122S1-CS2103T-T10-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code&authorshipIsBinaryFileTypeChecked=false)
+  - What it does: Deletes the specified Event.
```
Comment (lowkangn) [inline docs/team/davelowqx.md:20]: Same as above, try to write more if you can 👍

**Review by lowkangn — APPROVED** [review]
If you're really okay with how it is, I think it's good to merge. But try to write a bit more if you can.

File: docs/team/davelowqx.md (hunk 1 above)
Comment (davelowqx) [reply docs/team/davelowqx.md:15]: okok thanks bro will see what I can add on 😅😅😅

**Pull request merged**